tests/test_mkdocs_config.py

```
@@ -0,0 +1,611 @@
+    return config_path
+
+@pytest.fixture
+def minimal_config_file(temp_dir, minimal_mkdocs_config):
+    """Create a temporary mkdocs.yml file with minimal configuration."""
+    config_path = os.path.join(temp_dir, 'mkdocs_minimal.yml')
+    with open(config_path, 'w', encoding='utf-8') as f:
+        yaml.dump(minimal_mkdocs_config, f, default_flow_style=False)
+    return config_path
+
+# Configuration validation functions that would typically exist in the module
+def load_config(config_file_path: str) -> Dict[str, Any]:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)